### PR TITLE
LIME-405 - Increases SQS retention from 4 days to 7 days

### PR DESCRIPTION
**Pull request by MMcC-GDS**
### Issue tracking

- [LIME-405](https://govukverify.atlassian.net/browse/LIME-405)

[LIME-405]: https://govukverify.atlassian.net/browse/LIME-405?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: txma/template.yaml [hunk 1]
```diff
@@ -20,6 +20,9 @@ Parameters:
     ConstraintDescription: must be dev, build, staging, integration or production
   PermissionsBoundary:
     Type: String
+  SqsRetentionPeriod:
+    Type: Number
+    Default: 345600 # 4 days
```
Comment (merlinc) [inline txma/template.yaml:25]: Numbers like this are really difficult to eyeball and know that they are correct.

Could we implement something like https://github.com/alphagov/di-ipv-cri-kbv-api/blob/main/infrastructure/lambda/template.yaml#L147-L162 in order to capture the number and units separately?

File: .github/workflows/post-merge-publish-txma-infrastructure.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
   publish_txma_to_matrix_dev:
     strategy:
       matrix:
-        target: [ ADDRESS_DEV, FRAUD_DEV, KBV_DEV, KBV_POC, CIC_DEV ]
+        target: [ FRAUD_DEV ]
```
Comment (merlinc) [inline .github/workflows/post-merge-publish-txma-infrastructure.yaml:15]: Why are we removing all of these other environments?

**Review by merlinc — CHANGES_REQUESTED** [review]

File: .github/workflows/post-merge-publish-txma-infrastructure.yaml (hunk 2 above)
Comment (gds-jbentham) [reply .github/workflows/post-merge-publish-txma-infrastructure.yaml:15]: Only so we can check it doesnt cause any issues before pushing to all the CRIs

Comment (MMcC-GDS) [reply .github/workflows/post-merge-publish-txma-infrastructure.yaml:15]: At the moment, it is purely for testing purposes. This change is to be reverted.

File: txma/template.yaml (hunk 1 above)
Comment (gds-jbentham) [reply txma/template.yaml:25]: In that case we read the unit and the value in our code this time its an AWS config value so were limited to adhering to their formats

File: .github/workflows/post-merge-publish-txma-infrastructure.yaml (hunk 2 above)
Comment (mrdoodles) [reply .github/workflows/post-merge-publish-txma-infrastructure.yaml:15]: I think this is good just to test this in Fraud but it needs to be updated again to include the other environments once proven

**Review by mrdoodles — APPROVED** [review]

**Review by gds-jbentham — APPROVED** [review]

**Pull request merged**